buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -6,34 +6,34 @@ object Versions {
     internal const val ANDROID_GRADLE_PLUGIN = "7.3.1"
     internal const val ANDROID_JUNIT5_GRADLE_PLUGIN = "1.8.2.1"
     internal const val ANDROID_LEGACY_SUPPORT = "1.0.0"
-    internal const val ANDROIDX_ACTIVITY_COMPOSE = "1.6.1"
-    internal const val ANDROIDX_ACTIVITY_KTX = "1.6.1"
+    internal const val ANDROIDX_ACTIVITY_COMPOSE = "1.7.1"
+    internal const val ANDROIDX_ACTIVITY_KTX = "1.7.1"
     internal const val ANDROIDX_ANNOTATIONS = "1.5.0"
     internal const val ANDROIDX_APPCOMPAT = "1.5.1"
-    internal const val ANDROIDX_COMPOSE = "1.3.1"
-    public const val ANDROIDX_COMPOSE_COMPILER = "1.4.3"
+    internal const val ANDROIDX_COMPOSE = "1.4.1"
+    public const val ANDROIDX_COMPOSE_COMPILER = "1.4.4"
```

Comment:
It can't be upgraded to `1.4.5` yet because with this compiler version, gradle needs to use Java 17 and it is incompatible with our basecode.
They are proposing to downgrade it to Java 11 on version `1.4.6`

Reply:
![Screenshot from 2023-04-20 14-33-23](https://user-images.githubusercontent.com/4047514/233367095-865f3211-8db0-4422-97eb-cf8aeae484b2.png)
https://developer.android.com/jetpack/androidx/releases/compose-compiler#1.4.5
 
Because of this compiler, we can't upgrade Kotlin version yet :face_exhaling:

Reply:
New version 1.4.6 has been already released!!!! :partying_face: :tada: :tada: